tests/integration/suite/scheduler/api/http/errors.go

```diff
@@ -256,97 +256,4 @@ func (e *standardizedErrors) Run(t *testing.T, ctx context.Context) {
 		require.Equal(t, ErrInfoType, detailsObject["@type"])
 	})
 
-	// Covers DeleteJob job not found
```

Comment:
I think we still want this

Reply:
The current implementation, deleting an object that is not found is also a success. So, there is no error to test.

Reply:
Why are we making this design decision?

Reply:
Same as before: get and delete are not transaction, so there is no guarantee by doing the get upfront. Also, etcd accepts the delete as successful even if key does not exist.